packages/@webex/plugin-meetings/src/locus-info/embeddedAppsUtils.ts

```diff
@@ -52,7 +53,7 @@ EmbeddedAppsUtils.areSimilar = (apps1, apps2) => {
  * @param {array} embeddedApps
  * @returns {array} result - new array of parsed embedded app objects
  */
-EmbeddedAppsUtils.parse = (embeddedApps) =>
+EmbeddedAppsUtils.parse = (embeddedApps: Array<any>) =>
```

Comment:
@rajeshtezu90 `embeddedApps: Array<any> `can be replaced by `embeddedApps: any[]` if both are equivalent,

Reply:
This was picked by ts-migrate from jsDoc. Yes both are equivalent and we can update to `any[]` when we work on individual files as part of different tickets and adding proper types by removing `any`.